doc/sphinx/source/community/backward_compatibility.rst

```diff
@@ -0,0 +1,262 @@
+ESMValTool policy on backward compatibility
+===========================================
+
+V1.0. Agreed with ESMValTool community during October 2022 workshop.
```

Comment:
The version number here is slightly confusing to me. Why does this document need to be versioned independently of the rest of the tool?

Reply:
I agree. Line now removed.